Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -369,7 +370,17 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
         AppEnvironment.current.apiService.completeOnSessionCheckout(input: input).materialize()
       }
 
-    self.checkoutComplete = checkoutCompleteSignal.signal.values().ignoreValues()
+    let thanksPageData = Signal.combineLatest(initialData, baseReward)
+      .map { initialData, baseReward -> ThanksPageData? in
+        guard let reward = baseReward else { return nil }
+
+        return (initialData.project, reward, nil, initialData.total)
+      }
```

Comment:
It didn't look like we needed to pass any CheckoutData in this case so I passed nil instead.

Reply:
Huh. Do regular pledges require it, or can `CheckoutData` be cleaned up from the `ThanksPageData` type?

Reply:
Yea I have a note to look at that. I couldn't see find what it was needed for honestly.